packages/sit-onyx/src/components/OnyxIcon/types.ts

```diff
@@ -0,0 +1,16 @@
+import type { OnyxColor } from "@/index";
+
+export type OnyxIconProps = {
+  /**
+   * SVG source of the icon. **Important**: Only provide trustworthy content, the SVG content will
+   * not be sanitized.
```

Comment:
because we can not sanitize it or because sanitizing would make it too complicated / has side effects?